dll/win32/browseui/desktopipc.cpp

```diff
@@ -519,8 +523,13 @@ extern "C" void WINAPI SHDestroyIETHREADPARAM(IEThreadParamBlock *param)
         param->offset78->Release();
     if (param->offsetC != NULL)
         param->offsetC->Release();
+    // FIXME: This is completely broken!
+    // And I'm sure that there is a dedicated place
+    // in hell for people writing this kind of code!
+#ifndef _WIN64
     if (param->offsetF8 != NULL)
         param->offsetF8->Release();
+#endif
```

Comment:
How is it broken ? From a quick look, I don't see anything wrong.

Reply:
An interface pointer called `offsetF8` is wrong enough. Anyway, I debugged what is really wrong and fixed that.